cloudquery/sdk/schema/table.py

```diff
@@ -151,6 +151,8 @@ def filter_dfs_func(tt: List[Table], include, exclude, skip_dependent_tables: bo
     filtered_tables = []
     for t in tt:
         filtered_table = copy.deepcopy(t)
+        for r in filtered_table.relations:
```

Comment:
Related to https://github.com/cloudquery/cloudquery/issues/14637.
This was missing from the Python code and as a result the parent was never set. This means it was never sent via the gRPC call to get the tables.
See relevant Go code in https://github.com/cloudquery/plugin-sdk/blob/36128dd94c73675428b7380c9f7dccb0e6416758/schema/table.go#L201 and https://github.com/cloudquery/plugin-sdk/blob/36128dd94c73675428b7380c9f7dccb0e6416758/schema/table.go#L550